app/User.php

```diff
@@ -1954,37 +1952,14 @@ public static function getNextAvailableMemberId()
     /**
      * Find the lowest unused member id.
      *
-     * @TODO Refactor
+     * @deprecated Operation too expensive. Will now return the next available member id.
```

Comment:
Do we truly want to deprecate this, or keep it as a wrapper until we start pushing the need to start recycling, or come up with a better means of recycling the IDs (such as through a list we populate offline)?? I am in favor of keeping it, because while we might not need it ourselves right now, MEDUSA will need that functionality.